sr_fingertip_visualization/src/sr_fingertip_visualization/finger_widgets_graphs.py

```diff
@@ -273,17 +273,20 @@ def __init__(self, side, parent):
         layout = QVBoxLayout()
 
         layout.setAlignment(Qt.AlignVCenter)
-        no_tactile_label = QLabel("\t\tSTF sensors not supported yet."
-                                  " \nAlternatively, you can launch the PlotJuggler data inspection tool:")
+        no_tactile_label = QLabel("\t      STF sensors are not supported yet."
```

Comment:
mix of tabs and spaces seems odd

Reply:
Fixed now! https://github.com/shadow-robot/sr-visualization/pull/317/commits/f00ed1e6670662820df5d7b1aabfbaff73d30ca8